tests/workflow/workflow_with_models.py

```diff
@@ -66,6 +66,8 @@
 demo_workflow_run_copy = demo_workflow_run.copy()
 assert demo_workflow_run == demo_workflow_run_copy
 
+assert demo_workflow.method_dict('run', demo_workflow._method_jsonschemas['run']) != {}
```

Comment:
Technically, an empty dict could exist, since a instanciation could have no inputs

Reply:
i removed this line as there is a method dict unittest now